setup.cfg

```diff
@@ -1,13 +1,3 @@
-[nosetests]
-verbosity=2
-detailed-errors=1
-with-ignore-docstrings=1
-with-timer=1
-timer-top-n=15
-cover-package=trading_calendars
```

Comment:
Do we still have coverage running at all?

Reply:
I don't see where we were running or reporting on this information. Happy to run something under https://pypi.org/project/pytest-cov/ or https://coverage.readthedocs.io/en/latest/. 

If we do run coverage somewhere I'd like to report it somewhere actionable, not just as a log into the test suite.

Reply:
Maybe it's a relic from zipline, where we are or were using coveralls. Yep, feel free to add later, if we think it's useful.

Reply:
Follow up: https://github.com/quantopian/trading_calendars/issues/182